python/py-emcee/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        dfm emcee 2.2.1 v
+github.setup        dfm emcee 3.0.2 v
```

Comment:
I would recommend using the PyPI file directly, that also resolves the version issue I think. So you can remove the `github` PG and just set the `version` (see other Portfiles for examples).